lib/delivery_boy/config.rb

```diff
@@ -51,6 +51,7 @@ class Config < KingKonf::Config
     string :sasl_scram_username
     string :sasl_scram_password
     string :sasl_scram_mechanism
+    string :sasl_oauth_token_provider
```

Comment:
@dasch this variable will be set to an instance of a class. It works as expected using the `string`, however the King::Konf interface does not have a class definition it might look strange.

Reply:
In that case, use `attr_accessor :sasl_oauth_token_provider` instead. Otherwise, we will try to load the value from e.g. ENV or the YAML config.